admin-tools.Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN --mount=type=cache,target=/root/.cache/go-build (cd ./temporal && go mod dow
 COPY ./temporal ./temporal
 # Git info is needed for Go build to attach VCS information properly.
 # See the `buildvcs` Go flag: https://pkg.go.dev/cmd/go
-COPY ./.git ./.git
+# NOTE: `COPY ./.git ./.git` will fail in Docker as COPY isn't allowed to add git repositories
+COPY ./.gi? ./.git
```

Comment:
We could also use `.gi[t]` I believe.

Reply:
Let me get this out to unblock folks; and then I can try this one as a follow-up PR.